src/blocks/button.css

```
@@ -0,0 +1,17 @@
+.card__button-link {
+  align-items: center;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  background: rgba(0, 172, 220, 1);
+  margin-top: 16px;
+  text-decoration: none;
+  color: #fff;
+  text-transform: uppercase;
+  padding: 12px 55px;
+  transition: 1s;
```

You should specify the properties for transition
```suggestion
  transition: color 1s, background-color 1s;
```